components/NavBar/NavBar.tsx

```diff
@@ -61,22 +39,36 @@ const HeaderContentContainer = styled(Container)`
     margin-right: 5rem;
   }
 `
+const TopBarContainer = styled(TopBar)`
+  width: 100%;
+  padding-bottom: 4rem;  
 
+`
 const LogoContainer = styled.div`
   width: fit-content;
   white-space: nowrap;
 `
 
 const ActionsContainer = styled(TopBar.Actions)`
+  width: 100%;
```

Comment:
Same here, are you sure that we need all these custom styles or are they just something that was needed in the past?